src/app/components/availability/availability.directive.js

```diff
@@ -30,6 +30,7 @@ class AvailabilityController {
     this.changePasswordForm.newPasswordConfirm = "";
     this.showChangePassword = false;
     this.datepickerOptions = {
+      "minDate": new Date().setHours(0,0,0),
```

Comment:
Basically if you need to get a full timestamp use `this.moment().startOf("day").valueOf()`